src/cr/cube/cube.py

```diff
@@ -263,6 +262,12 @@ def has_means(self):
         """True if cube includes a means measure."""
         return self._measures.means is not None
 
+    @lazyproperty
+    def means_subvariables(self):
+        if not self.has_means:
+            return None
+        return self._measures.means.subvariables
+
```

Comment:
Also this property should be right before the `missing` one. just to respect the alphabetical oder

Reply:
This name `means_subvariables` is odd to me 🤷‍♂️  , can you explain me what's a subvariable of means? I mean i got from the code that the subvariables belongs to a variable and they're than retrieved in the MeanMeasure, but i don't get te sense of this name ... sorry